torch/csrc/lazy/ts_backend/ts_backend_impl.cpp

```diff
@@ -40,7 +39,7 @@ class TSBackendImpl : public torch::lazy::BackendImplInterface {
   TSBackendImpl() : default_device_type_(at::kCPU) {
     // TODO(whc) unify how all our flags are set and parsed as envs
     static bool env_use_cuda = std::getenv("LTC_TS_CUDA") != nullptr;
-    auto type = (env_use_cuda || FLAGS_torch_lazy_ts_cuda) ? at::kCUDA : at::kCPU;
+    auto type = (env_use_cuda || FLAGS_torch_lazy_tensors_cuda) ? at::kCUDA : at::kCPU;
```

Comment:
this seems wrong to me; i think this flag is clearly a ts-backend flag not a general one